pyproject.toml

```diff
@@ -63,7 +63,7 @@ requires = ["setuptools>=61", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
-packages = ["pan3d", "pan3d.ui", "pan3d.catalogs"]
+packages = ["pan3d", "pan3d.ui", "pan3d.ui.pan3d_components", "pan3d.ui.pan3d_components.module", "pan3d.catalogs"]
```

Comment:
Is just listing "pan3d" not enough?

Reply:
In my experience, this results in an import error when attempting to import something in any constituent folders.

Reply:
For example, https://github.com/Kitware/geo-trame/actions/runs/9570591491/job/26385698536 failed prior to [8389496](https://github.com/Kitware/geo-trame/pull/78/commits/8389496ac09e0b4988c4d733fd6ac0c91f34d3bf)